streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -662,6 +664,16 @@ object StreamingContext extends Logging {
     }
   }
 
+  /**
+   * Get the context that is currently active, that is, started but not stopped.
+   */
+  @Experimental
+  def getActive(): Option[StreamingContext] = {
+    ACTIVATION_LOCK.synchronized {
+      Option(activeContext.get())
```

Comment:
I am updating my the branch, but even without the update, this seems fine to me. ACTIVATION_LOCK is used for start() in this branch already.